duply-bucket-iamuser/variables.tf

```diff
@@ -7,3 +8,4 @@ variable "project" {
 variable "environment" {
   default = "production"
 }
+
```

Comment:
can you define the types on these

Reply:
should be done, If I'm not mistaken (looking at other updated modules), we only define types when there's no default or when type is complex, no?

Reply:
with the new complex variable types, I think it's a best practice to enforce the type of all variables, even if they have defaults.